src/test/modelmanager_test.cpp

```diff
@@ -270,6 +270,28 @@ static const char* modelMetricsMissingPort = R"(
         }
 })";
 
+static const char* modelMetricsMissingPortWithDisabledMetrics = R"(
+{
+    "model_config_list": [
+        {
+            "config": {
+                "name": "dummy",
+                "base_path": "/ovms/src/test/dummy",
+                "target_device": "CPU",
+                "model_version_policy": {"latest": {"num_versions":1}},
+                "nireq": 100,
+                "shape": {"b": "(1,10) "}
+            }
+        }
+    ],
+    "monitoring":
+        {
+            "metrics":
+            {
+                "enable" : false
```

Comment:
It seems like we only handle `--metrics_enable`. Metrics list should also be accepted via CLI.